### ISSUE TITLE
README.md says no access to 'std' for guest

### ISSUE THREAD

**Issue by stskeeps** [risc0/risc0-rust-starter#17]
.. but it does have?

**Comment by KuTuGu** [comment]
I think this will help
```
risc0-zkvm = { version = "0.12", features = [ "std" ] }
```

**Comment by tzerrell** [comment]
On the guest side, you also (always) need `default-features = false`. So
```
risc0-zkvm = { version = "0.12", default-features = false, features = [ "std" ] }
```
is how you enable `std` in the guest.

As for the misleading documentation, we fixed that back in #20, so this issue is resolved.
